utils/pathHelper.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2025 The Kanister Authors.
```

Comment:
Just to be consistent with the format used for file names across the repo - please change the file name format to `path-helper.go`